src/platform_api/ios.py

```diff
@@ -56,11 +56,14 @@ def build_analytics_order(email, aois, schedule, workflow_request, cd_params=Non
     if cd_params is not None:
         req["changeDetAnalyticParams"] = cd_params
 
-    return build_order(req)
+    return build_order(req, customer_notes)
 
 
-def build_order(request):
-    return {"customerNotes": "#ESRI-TOOLBOX", "request": request}
+def build_order(request, customer_notes=None):
+    if customer_notes:
```

Comment:
We should probably also check if customer notes equals an empty string (like ' ') and not add the customer notes field if the user entered a random space.

when i don't submit a note at all, the jira ticket looks like this (no customer notes field present) https://ursaspace.atlassian.net/browse/ORD-312

but when i enter ' ' in the customer notes field, the ticket looks like (with customer note field present but nothing in it)
https://ursaspace.atlassian.net/browse/ORD-311

I think we should probably handle empty strings the same way we handle no input

Reply:
I agree, I just added a check for this. If the note is only spaces (one or more) no note will be added, if the note contains spaces and character the note will be added.